modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/NewTopicModal.es.js

```diff
@@ -133,33 +140,35 @@ export default function NewTopicModal({
 									ref={topicDescriptionRef}
 								/>
 							</ClayForm.Group>
-						</ClayForm>
-					</ClayModal.Body>
-
-					<ClayModal.Footer
-						last={
-							<ClayButton.Group spaced>
-								<ClayButton
-									aria-label={Liferay.Language.get('cancel')}
-									displayType="secondary"
-									onClick={close}
-								>
-									{Liferay.Language.get('cancel')}
-								</ClayButton>
-
-								<ClayButton
-									aria-label={Liferay.Language.get('create')}
-									displayType="primary"
-									onClick={() => {
-										createTopic();
-										close();
-									}}
-								>
-									{Liferay.Language.get('create')}
-								</ClayButton>
-							</ClayButton.Group>
-						}
-					/>
+						</ClayModal.Body>
+
+						<ClayModal.Footer
+							last={
+								<ClayButton.Group spaced>
+									<ClayButton
+										aria-label={Liferay.Language.get(
+											'cancel'
+										)}
+										displayType="secondary"
+										onClick={close}
+									>
+										{Liferay.Language.get('cancel')}
+									</ClayButton>
+
+									<ClayButton
+										aria-label={Liferay.Language.get(
+											'create'
+										)}
+										displayType="primary"
+										onClick={handleSubmit}
```

Comment:
You don't need `onClick` this automatically work, because the button is type submit and is inside a form.

Reply:
That is true, I'll change that

Reply:
Removed in https://github.com/liferay-content-management/liferay-portal/pull/5366/commits/c047e0b2b13396025cb4c8bd62b146e0ae52b90a